doc/tutorial/erda.rst

```diff
@@ -0,0 +1,185 @@
+
+Below the description there is a **Start SERVICE** button, which you can click to open a connection to that particular service in a new web browser tab or window.
+
+.. figure:: /_images/erda/erda_dag_spawn.png
+   :width: 100%
+   :align: center
+
+   Select a Jupyter Service menu.
+
+By default, it will take you to your personal home page on the **Jupyter service** as shown below, which is provided via a hosted version of JupyterHub.
+That is, the standard infrastructure to provide individual isolated Jupyter notebook containers to multiple users sharing a pool of actual compute nodes.
+
+.. figure:: /_images/erda/erda_jservice_homepage.png
+   :width: 100%
+   :align: center
+
+   Top fragment of Jupyter service home page.
+
+After clicking **Start My Server**, the site will give you an option to chose which notebook image you want to spawn.
+Select **HPC Notebook** as shown below and press the **Start** button.
+
+.. figure:: /_images/erda/erda_dag_image.png
+   :width: 100%
+   :align: center
+
+   Top fragment of Jupyter service home page with selected HPC Notebook image.
+
+This will spawn the **HPC Notebook** image and redirect you straight to the JupyterLab interface as shown below.
+The JupyterLab interface is the same in all available Services (DAG and MODI).
+
+.. figure:: /_images/erda/erda_dag_terminal.png
+   :width: 100%
+   :align: center
+
+   JupyterLab interface on DAG.
+
+Follow the Veros installation instructions below with respect to the selected service.
+
+Data Analysis Gateway (DAG)
++++++++++++++++++++++++++++
+
+In order to install Veros on a DAG instance do the following after launching the **Terminal**:
+
+1. Clone the Veros repository
+
+   .. exec::
+
+      from veros import __version__ as veros_version
+      if "+" in veros_version:
+          veros_version, _ = veros_version.split("+")
+      print(".. code-block::\n")
+      print(f"   $ git clone https://github.com/team-ocean/veros.git -b v{veros_version}")
+
+   (or `any other version of Veros <https://github.com/team-ocean/veros/releases>`__).
+
+2. Change the current directory to the Veros root directory::
+
+      $ cd veros
+
+3. Create a new conda environment for Veros, and install all relevant dependencies by running::
+
+      $ conda env create -f conda-environment.yml
+
+4. To use Veros, activate your new conda environment via::
+
+      $ conda activate veros
+
+5. Make a folder for your Veros setups, and switch to it::
+
+      $ mkdir ~/vs-setups
+      $ cd ~/vs-setups
+
+6. Copy the :doc:`global 4deg </reference/setups/4deg>` model template from the :doc:`setup gallery </reference/setup-gallery>`::
+
+      $ veros copy-setup global_4deg
+
+7. Change the current directory to the setup directory::
+
+      $ cd global_4deg/
+
+8. Modify model parameters with the `nano text editor <https://www.nano-editor.org>`__::
```

Comment:
Wouldn't it be easier to find the file in the sidebar and edit it in the built-in JupyterLab editor (double click)?

Reply:
Yes.